Cargo.toml

```diff
@@ -42,8 +42,9 @@ version = "0.2"
 name = "rustpython"
 path = "src/main.rs"
 
-# Uncommment when you want to compile/check with redoxer
-# [patch.crates-io]
+[patch.crates-io]
+# REDOX START, Uncommment when you want to compile/check with redoxer
```

Comment:
Maybe a patch file on `Cargo.toml` in the redox folder might be a better option here?

Reply:
If it were a patch file, it would only work with the same number of lines; it would break if we were to add more lines e.g. dependencies above where it's switching the lines.

Reply:
I would prefer doing a target dependent dependency like in the `vm` Cargo.toml. Some thing like
`[target.'cfg(not(target_os = "redox"))'.dependencies]` and `[target.'cfg(target_os = "redox")'.dependencies]`

Reply:
I tried doing `target.cfg().patch` syntax at first, but it didn't work for some reason.

Reply:
Why do we need to use the `target.cfg().patch` and not `target.cfg().dependencies`?

Reply:
This might be a nice option, so we can use feature flags, or even let cargo figure out the os, and use the `target_os` flag.